.drone.star

```diff
@@ -1711,6 +1668,112 @@ def redisService():
 		},
 	}]
 
+def cacheOcisPipeline(ctx):
+	return[{
+		'kind': 'pipeline',
+		'type': 'docker',
+		'name': 'cache-ocis',
+		'workspace' : {
+			'base': '/var/www/owncloud',
+			'path': config['app']
+		},
+		'steps':
+			buildOCISCache() +
+			cacheOcis() +
+			listRemoteCache(),
+		'depends_on': [],
+		'trigger': {
+			'ref': [
+				'refs/pull/**',
+			]
+		}
+	}]
+
+def getOcis():
+	return [{
+		'name': 'get-ocis-from-cache',
+		'image': 'minio/mc:RELEASE.2020-12-10T01-26-17Z',
+		'failure': 'ignore',
+		'environment': {
+			'MC_HOST': {
+				'from_secret': 'cache_s3_endpoint'
+			},
+			'AWS_ACCESS_KEY_ID': {
+				'from_secret': 'cache_s3_access_key'
+			},
+			'AWS_SECRET_ACCESS_KEY': {
+				'from_secret': 'cache_s3_secret_key'
+			},
+		},
+		'commands': [
+			'source /var/www/owncloud/web/.drone.env',
+			'mkdir -p /var/www/owncloud/ocis-build',
+			'mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY',
+			'mc mirror s3/owncloud/web/ocis-build/$OCIS_COMMITID /var/www/owncloud/ocis-build/',
+			'chmod +x /var/www/owncloud/ocis-build/ocis',
+		]
+	}]
+
+def buildOCISCache():
+	return[{
+		'name': 'build-ocis',
+		'image': 'webhippie/golang:1.16',
+		'pull': 'always',
+		'commands': [
+			'./tests/drone/cache_ocis.sh'
```

Comment:
```suggestion
			'./tests/drone/build_ocis.sh'
```
Maybe this script would be better called `build_ocis.sh` ? Or?
It seems to build an OCIS - other things put that in the cache or get it back.